azurerm/resource_arm_network_interface_test.go

```diff
@@ -182,6 +182,25 @@ func TestAccAzureRMNetworkInterface_enableIPForwarding(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMNetworkInterface_enableAcceleratedNetworking(t *testing.T) {
```

Comment:
can we also add [a test that provisions a VM using a Managed Disk in this file](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_virtual_machine_managed_disks_test.go)? That would allow us to have an end-to-end example of this working

Reply:
Sure can. I'll also rebase against the latest master.

Reply:
The one caveat here is that only certain instance types support AN. I will select the smallest I know of, which is a `Standard_D8_v3`, but it is not an inexpensive machine.

Reply:
Given the acceptance tests only spin the machines up for tests for a short while, I don't think that'll be that bad for us (HashiCorp) - however I think it's worth mentioning for contributors. In the AWS Provider we document this kind of thing at the top of the file, for instance:

```
// NOTE: Test `X` requires a machine of size `D8_v3` which is large/expensive - you may wish to ignore this test"
```

As such - can we update the top of the `resource_arm_virtual_machine_managed_disk_test.go` file with a comment to this effect?

Reply:
@tombuildsstuff I put that test in the wrong file.... it's in the network interface tests. I'll move it over now and update the top as requested.